### ISSUE TITLE
Links not accessible (a11y)

### ISSUE THREAD

**Issue by filipjnc** [supertokens/supertokens-core#483]
## 🐛 Bug Report

All the links in the UI are not accessible. 

## Useful informations

I recommend switching to <a> for links, as most accessibility is achieved out of the box with it. Following is the bare minimum if you want to continue with div/span elements:
- Link markup (`role="link"`)
- Keyboard-focusable
- Keyboard activation
- Focus order (`tabindex=0`)
- Focus indicator
